activerecord/test/cases/invertible_migration_test.rb

```diff
@@ -526,5 +532,22 @@ def test_up_only
       assert_not connection.column_exists?(:horses, :oldie)
       Horse.reset_column_information
     end
+
+    def test_down_only
+      InvertibleMigration.new.migrate(:up)
+      horse1 = Horse.create
+      # populates existing horses with oldie = 1 but new ones have default 0
+      UpOnlyMigration.new.migrate(:up)
```

Comment:
Could you change this test to use DownOnlyMigration here as well?
This would make sure it's only run on up.
Maybe something like:
```ruby
   class DownOnlyMigration < SilentMigration
      def change
        down_only { execute "update horses set remind_at = '2030-01-01'" }
      end
    end
```